Roomz/src/main/webapp/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Roomiez</title>
+    <!-- W3 Schools Stylesheets -->
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-green.css">
+    <!-- Javascript file -->
+    <script type="text/javascript" src="script.js"></script>
+</head>
+
+<body>
+    <div>
+        <!-- NAVIGATION BAR -->
+        <header>
+            <nav class="w3-bar w3-theme-d5 w3-padding">
+                <a href="#" class="w3-bar-item w3-button">ROOMIEZ</a>
```

Comment:
OOC, is the project called Roomz or Roomiez? haha

Reply:
haha actually we are still working on the name. Those are the two names I came up with. Would you suggest adding a more generic folder name such as "team project" or should I leave it like that for now?

Reply:
nope, it's fine for the code directory name and the final name to be different. Just curious for now